packages/metamask/README.md

```diff
@@ -0,0 +1,51 @@
+# @web3-onboard/metamask
+
+## Wallet module for connecting MetaMask Wallet SDK to web3-onboard
+See [MetaMask SDK Developer Docs](https://github.com/MetaMask/metamask-sdk)
```

Comment:
It would be good to explain a little more here about what can be done with the SDK, connect mobile MM to desktop dapps, etc
I will get any additional verbiage added to our docs pages as well!